src/components/MegaButton/__tests__/MegaButton.spec.js

```diff
@@ -161,7 +195,107 @@ describe('Megabutton', () => {
       });
 
       const text = getByText(props.text);
-      expect(text).toBeValid();
+      expect(text).not.toBeInTheDocument;
+    });
+
+  });
+
+  describe('Small text with image', () => {
+
+    const textWithImageProps = {
+      id: 'megabutton',
+      modelValue: '',
+      name: 'Fine print',
+      label: 'Megabutton with text',
+      text: 'This also has an image',
+      imageSource: 'fake_image.egg',
+      value: 'bigOlButton'
+    };
+
+    it('renders the label in smaller text', () => {
+      const props = textWithImageProps;
+
+      const { getByText } = render(MegaButton, {
+        props
+      });
+
+      const label = getByText(props.label);
+      expect(label).toHaveClass('text-lg');
+    });
+
+    it('renders the text', () => {
+      const props = textWithImageProps;
+
+      const { getByText } = render(MegaButton, {
+        props
+      });
+
+      const text = getByText(props.text);
+      expect(text).not.toBeInTheDocument;
```

Comment:
I don't believe this is correct, it's a function, it should have the parens: https://github.com/testing-library/jest-dom#tobeinthedocument.  This makes me wonder if there's something wrong with the tests at a higher level, this should be failing.

Reply:
argh, you're right. I missed those other two occurences.